pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -672,6 +686,18 @@ func getVolumesFromPodDir(podDir string) ([]podVolume, error) {
 			}
 		}
 	}
+	mps, err := mounter.List()
+	if err != nil {
+		glog.Errorf("Could not get mount paths")
+	}
+	for i := range mps {
+		for j := range volumes {
+			volume := &volumes[j]
+			if mps[i].Path == volume.mountPath {
+				volume.devicePath = mps[i].Device
```

Comment:
Can you explain what this code is doing?

Reply:
The code is trying to check what is the device for each mountpath in the volumes. First we get the mountpaths and their devices from /proc/mount, and then match the mountpath with the volumes to get the device name.

Reply:
Ack.